shard-operator/src/main/kubernetes/minikube.yml

```diff
@@ -24,6 +24,7 @@ rules:
       - deployments
       - services
       - namespaces
+      - configmaps
```

Comment:
@r00ta Sorry if I'm missing something obvious here. 

I think you're binding the Processor definition as an environment variable on the Deployment container. Is there something else that means the Shard needs permissions on `ConfigMaps`?

Reply:
Hi @robpblake , yep this is a leftover. No need anymore for permissions on configmaps